utf8.js

```diff
@@ -151,13 +151,13 @@
 		byteIndex++;
 
 		// 1-byte sequence (no continuation bytes)
-		if ((byte1 & 0x80) == 0) {
+		if ((byte1 & 0x80) === 0) {
 			return byte1;
 		}
 
 		// 2-byte sequence
 		if ((byte1 & 0xE0) == 0xC0) {
-			var byte2 = readContinuationByte();
+			byte2 = readContinuationByte();
```

Comment:
I’d prefer to keep the `==` the way they are. Could you submit this one (dropping the redundant `var`) as a separate patch please?